contracts/strategies/traderjoe/TraderJoe.sol

```diff
@@ -31,47 +30,135 @@ contract TraderJoe is UUPSUpgradeable, StrategyOwnablePausableBaseUpgradeable {
 
     function initialize(
```

Comment:
have you thought of calling initialize from 'reinitialize'? That would be a good way to reuse the same code

Reply:
initializer modifier

Reply:
are you suggesting adding a new modifier to share code?

Reply:
reinitialize has reinitializer(2) so it can't call initilize

Reply:
but you can move all the common code to a third funtion and call that function from both initialize and reinitialize, no?